services/rds/src/main/java/org/finra/gatekeeper/services/auth/GatekeeperRoleService.java

```diff
@@ -181,4 +225,64 @@ private GatekeeperRdsRole checkGatekeeperRdsRole() {
             return GatekeeperRdsRole.UNAUTHORIZED;
         }
     }
+
+
+
+    public Map<String, Map<String, Map<String, Integer>>> getApprovalPolicy(Map<String, RoleMembership> roleMemberships) {
+        Map<String, Map<String, Map<String, Integer>>> approvalPolicy = new HashMap<>();
+        for(String application : roleMemberships.keySet()) {
+            approvalPolicy.put(application, getApprovalPolicyByApplication(roleMemberships.get(application)));
+        }
+        return approvalPolicy;
+    }
+
+    private Map<String, Map<String, Integer>> getApprovalPolicyByApplication(RoleMembership roleMembership) {
+        gatekeeperRoles = new ArrayList<>();
+        gatekeeperRoles.add("datafix");
+        gatekeeperRoles.add("readonly_confidential");
+        gatekeeperRoles.add("readonly");
+        gatekeeperRoles.add("dba_confidential");
+        gatekeeperRoles.add("dba");
+
+        sdlcs = new ArrayList<>();
+        sdlcs.add("dev");
+        sdlcs.add("qa");
+        sdlcs.add("prod");
```

Comment:
Same problem down here, there could be users who are using SDLC values that aren't "dev", "qa", or "prod" we need to get the list of unique SDLC's from the provided configurations. See this class: 

https://github.com/FINRAOS/Gatekeeper/blob/master/services/rds/src/main/java/org/finra/gatekeeper/configuration/GatekeeperApprovalProperties.java 

and this class:

https://github.com/FINRAOS/Gatekeeper/blob/master/services/rds/src/main/java/org/finra/gatekeeper/configuration/GatekeeperOverrideProperties.java